pages/curriculum.tsx

```diff
@@ -26,9 +26,30 @@ const Curriculum: React.FC = () => {
     const { lessons }: { lessons: Lesson[] } = data
     const sortedLessons: React.ReactElement[] = lessons
       .sort((a, b) => a.order - b.order)
-      .map((e: Lesson) => (
+      .map((e: Lesson, idx: number) => (
         <Card key={e.id} title={e.title}>
-          <p>{e.challenges.length} Challenges</p>
+          <img
+            src={`/curriculumAssets/lessonCoversSvg/js-${idx}-cover.svg`}
+            alt={`js${idx}-cover`}
+          />
```

Comment:
If you want you can do something more sexy by importing the SVG
```
import MyImage from './myImage.svg'
  <div>
    <MyImage />
  </div>
```
[Doc about it](https://react-svgr.com/docs/webpack/)